### PR TITLE
update japanese translate

### PR DESCRIPTION
**Pull request by westooooo**

### CODE REVIEW

File: lua/terrortown/lang/ja.lua [hunk 1]
```diff
@@ -1408,425 +1404,429 @@ L.label_shop_linker_set = "ショップ設定"
...
+L.help_prop_possession = [[
+オブジェクト憑依は、観戦者がマップに存在するオブジェクトに憑依し、
+ゆっくりとチャージされていく「パンチ・オー・メーター」を使用して、そのオブジェクトを操作できる機能です。
+「パンチ・オー・メーター」の最大値は、2つの定義された制限の間に遮断された死量/死の差が追加される基本的憑依値で構成されています。
+メーターは時間の経過とともにゆっくりチャージされます。セットの再チャージ時間は、「パンチ・オー・メーター」の単一ポイントをチャージするのに必要な時間です。]]
+L.help_karma = "カルマは無差別殺害を減らすための機能です。プレイヤーは一定のカルマから始まり、チームメイトにダメージを与える、又は殺すと減っていきます。\n減少量は、与えたダメージ量や、殺害数に応じて変化します。カルマが少ないほど与えるダメージが減っていきます。"
+L.help_karma_strict = "厳密なカルマが有効になっている場合、カルマが減少するとダメージペナルティが増加します。無効の場合、\nプレイヤーがカルマ値800を超えていると、ダメージペナルティはかなり低くなります。厳密なカルマモードを有効にすると、\nカルマは不必要なキルを阻止する上でより大きな役割を果たし、それを無効にすると、カルマは常にチームキルするプレイヤーを傷つけるだけの緩いゲームになります。"
+L.help_karma_max = "1000以上の最大値カルマの値を設定しても、その1000カルマ以上のプレイヤーにダメージボーナスは与えられません。カルマバッファーとして使用できます。"
+L.help_karma_ratio = "両者が同じ陣営にいる場合に、カルマを加害者から差し引く量を計算するためのダメージ比率。
+チームキルが発生した場合は、さらにペナルティが適用されます。"
```
Comment (saibotk) [inline lua/terrortown/lang/ja.lua:1712]:
```suggestion
L.help_karma_ratio = "両者が同じ陣営にいる場合に、カルマを加害者から差し引く量を計算するためのダメージ比率。チームキルが発生した場合は、さらにペナルティが適用されます。"
```

**Review by saibotk — CHANGES_REQUESTED** [review]

File: lua/terrortown/lang/ja.lua [hunk 2]
```diff
@@ -1408,425 +1404,428 @@ L.label_shop_linker_set = "ショップ設定"
...
+L.label_lastwords_chatprint = "タイピング中に殺されたら遺言を送信する"
+L.label_identify_body_woconfirm = "'確認'ボタン無しで死体を特定"
+L.label_announce_body_found = "死体発見時の報告"
+L.label_confirm_killlist = "確認済みの死体のリストの報告"
+L.label_inspect_detective_only = "死体検査を確白役職にしかできないようにする"
+L.label_confirm_detective_only = "死体確認を確白役職にしかできないようにする"
+L.label_dyingshot = "アイアンサイト中で死に撃つ[実験的]"
+L.label_armor_block_headshots = "ヘッドショットへの耐久"
+L.label_armor_block_blastdmg = "爆破ダメージへの耐久"
+L.label_armor_dynamic = 動的アーマー"
```
Comment (saibotk) [inline lua/terrortown/lang/ja.lua:1774]:
```suggestion
L.label_armor_dynamic = "動的アーマー"
```

**Review by saibotk — CHANGES_REQUESTED** [review]

File: lua/terrortown/lang/ja.lua (hunk 2 above)
Comment (westooooo) [reply lua/terrortown/lang/ja.lua]: oh... I missed it... Thanks!

**Comment by westooooo** [comment]
I fixed it.

**Review by saibotk — APPROVED** [review]
Thank you very much 🥳

**Review by ZenBre4ker — APPROVED** [review]
No clue about japanese. Wont check them with google translator 😆 But structure looks alright. Thank you.

**Pull request merged**